jewelbots_nRF51822_board_v100/cores/JWB_nRF51822/Timer.cpp

```diff
@@ -56,5 +61,10 @@ extern "C"{
     }
   }
 
+  uint32_t Timer::millis(void) {
+    uint32_t num_ticks;
+    app_timer_cnt_get(&num_ticks); // TODO What to do with returned status?
```

Comment:
Hi @olore, thanks for pointing out the documentation of `app_timer_cnt_get`. I did see it, but I was not sure how to use it.

I will look into the code for the `button_handler.c` some more to see what the `APP_ERROR_CHECK` function does. Maybe it suffices to just call that.